claat/render/template.go

```diff
@@ -27,11 +27,12 @@ import (
 
 // Context is a template context during execution.
 type Context struct {
-	Env      string
-	Prefix   string
-	GlobalGA string
-	Meta     *types.Meta
-	Steps    []*types.Step
+	Env       string
+	Prefix    string
+	GlobalGA  string
+	Meta      *types.Meta
+	Steps     []*types.Step
+	ExtraVars map[string]string // Extra variables passed from the command line.
```

Comment:
Done making it Extra. I think map[string]string is fine. It is all we likely need ever, easier to use, and it is easy to update to interface if we ever do need it.